### PR TITLE
5 19 make sure employer cashflow to email works

### PR DESCRIPTION
**Pull request by YCChenVictor**

### CODE REVIEW

**Review by huihsinhsiao — APPROVED** [review]
👍

File: app/views/user_mailer/someone_apply_note.html.erb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <%= @employer.name %>，你好：
-
-有人應徵您的職缺
-
-如下：<%= @applicant.id %>
-
+<br>
+有人應徵您的職缺，如下：
+<br>
+<%= @applicant.id %>
```
Comment (huihsinhsiao) [inline app/views/user_mailer/someone_apply_note.html.erb:5]: 這裡要印出id還是印出name比較好呀～

**Review by yuRicky09 — APPROVED** [review]
nice

**Review by hsufenglin — APPROVED** [review]
👍👍

**Pull request merged**